<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultForm</class>
 <widget class="QDialog" name="ResultForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>691</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>691</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Результат</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/logo.png</normaloff>:/icons/icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color: #212630;
	color: #CFD2DC;
	border-color: #1C2227;

}

QFrame
{
	background-color: #212630;
	border: 1px solid #212630;
	border-radius: 10px;
}


/*-----QComboBox-----*/
QComboBox
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #232935;
	color: #CFD2DC;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #353C4F;

}


QComboBox:on
{
    background-color: #3BB0D7;

}


QComboBox QAbstractItemView
{
    background-color: #1C2227;
    color: #CFD2DC;
    border: 1px solid black;
    selection-background-color: #3BB0D7;
    selection-color: #CFD2DC;
    outline: 0;

}


QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;

}


QComboBox::down-arrow
{
    image: url(:/icons/icons/arrow-down.png);
    width: 8px;
    height: 8px;
}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #232935;
	color: #CFD2DC;
	font-weight: bold;
	border-style: solid;
	border-width: 2px;
	border-radius: 20px;
	border-color: #3BB0D7;
	padding: 10px;
	highlited:true;
}


QPushButton::hover
{
	background-color: #353C4F;
	color: #CFD2DC;

}


QPushButton::pressed
{
	background-color: #3BB0D7;
	color: #CFD2DC;

}

/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	background-color: #232935;
	color: #CFD2DC;
	border: 2px solid #3BB0D7;
	border-radius: 10px;
	padding : 2px;

}



QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #232935;
    border: 2px solid #6CB2FF;
    color:  #CFD2DC;
    padding: 2px

}


QSpinBox::up-button,
QDoubleSpinBox::up-button,
QDateTimeEdit::up-button
{
    background-color: #3BB0D7;
	border-top-right-radius: 5px;
	border-top-left-radius: 1px;

}

QSpinBox::down-button,
QDoubleSpinBox::down-button,
QDateTimeEdit::down-button
{
    background-color: #3BB0D7;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 1px;

}

QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #45CDFA;
    border: 2px solid #45CDFA;

}

QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #CFD2DC;
    border: 2px solid #CFD2DC;

}

QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow

{
    image: url(:/icons/icons/arrow-down.png);
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow

{
    image: url(:/icons/icons/arrow-up.png);
    width: 7px;

}



</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Результат вычислений</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>651</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма на конец срока:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_value_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_res">
      <property name="minimumSize">
       <size>
        <width>267</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
QLabel
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_value_1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Изначальная сумма: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма процентов к выплате: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_value_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_old">
      <property name="minimumSize">
       <size>
        <width>267</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
QLabel
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_difference">
      <property name="minimumSize">
       <size>
        <width>267</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
QLabel
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
